Imagehandler.py

```diff
@@ -7,6 +7,7 @@
 from PatternFinding import PatternFinding
 from FindingOrientationOfContours import FindingOrientationOfContours
 from AffineTransformation import AffineTransformation
+import numpy as np
```

Comment:
are we using this module in this file?

Reply:
In case of debugging, line no - 70 uses numpy. But we are not using numpy.